dp/generate_prsn.py

```diff
@@ -19,24 +19,36 @@
 logger = logging.getLogger(__name__)
 
 
-def dry_run(filepaths):
+def dry_run(filepaths, output_to_file=False, workdir=None):
     '''Perform metadata checks on the input files'''
```

Comment:
You may want to consider adding a docstring here since the method has some new options others may not understand.